api/src/main/java/org/eclipse/microprofile/config/Config.java

```diff
@@ -46,6 +46,10 @@
  * <p>
  * The config objects produced via the injection model <pre>@Inject Config</pre> are guaranteed to be serializable, while
  * the programmatically created ones are not required to be serializable.
+ * <p>
+ * If one or more converters are registered for a class of a requested value then one of the registered converters
```

Comment:
It should say the Converter with the high priority will be used to convert the string.

Reply:
I had a question here too.  In English, one's "highest priority" is also one's "first" priority (i.e. a "high priority" has a low number, and a "low priority" has a high number).  I assume "high priority" here means simply "greatest `value()` element of the `@Priority` annotation?

Reply:
The highest priority is the one with the high number.